roles/cloud-hetzner/tasks/main.yml

```diff
@@ -1,20 +1,17 @@
 ---
-- name: Build python virtual environment
-  import_tasks: venv.yml
-
```

Comment:
Why was this removed together with the `venv.yml`? While testing this I could not proceed with the algo configuration because of various missing python libs, however the error went away after re-introducing these lines.

Reply:
One of the error was:
```
TASK [cloud-hetzner : Get regions] *********************************************************************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to import the required Python library (python-dateutil) on .. Python projects/algo/.env/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}
```

Reply:
Well yeah, apparently we still need the Python libraries. I mistakenly assumed that they come with the collection.